proto/warden/warden/v1beta3/signature.proto

```diff
@@ -27,6 +27,20 @@ enum SignRequestStatus {
   SIGN_REQUEST_STATUS_REJECTED = 3;
 }
 
+// SignRequestType indicates the type of a signature request.
+//
+// This specifies how the transaction should be broadcasted.
+enum SignRequestType {
+  // The request is missing the type field.
+  SIGN_REQUEST_TYPE_UNSPECIFIED = 0;
+
+  // The transaction should be broadcasted immediately by the frontend or client.
+  SIGN_REQUEST_TYPE_IMMEDIATE_BROADCAST = 1;
+
+  // The transaction is to be broadcasted automatically by a scheduler without manual intervention.
+  SIGN_REQUEST_TYPE_SCHEDULED_BROADCAST = 2;
+}
+
```

Comment:
I don't like the terms "type", "immediate" and "scheduled" to indicate who is going to consume this signature.

I'd do it like this:

```suggestion
enum BroadcastType {
  // The signature should not be broadcasted.
  BROADCAST_TYPE_DISABLED = 0;
  
  // The signature should be automatically broadcasted by an offchain relayer.
  BROADCAST_TYPE_AUTOMATIC = 1;
}
```

which is also backward compatible.

Note that these protobufs are the source of truth for the protocol itself, mentioning a "frontend" here is not ideal, as there will be many different types of client and we don't really care if it's a browser or another kind of services

Reply:
I really like your suggestion, but I have a follow-up question: most other enums have 'unspecified' value, so BroadcastType will be an exception? But due to this, we won't need a migration? I guess consistency can be traded here for simplicity, just making sure I understand you correctly